scripts/mirrorImages.sh

```diff
@@ -5,15 +5,9 @@ set -euo pipefail
 declare -A imagearray=()
 imagearray["gcr.io/kpt-functions/read-yaml"]="gcr.io/config-management-release/read-yaml"
 imagearray["gcr.io/kpt-functions/write-yaml"]="gcr.io/config-management-release/write-yaml"
-imagearray["gcr.io/kpt-functions/gatekeeper-validate"]="gcr.io/kpt-functions/gatekeeper-validate"
```

Comment:
this is the important function that was previously being mirrored. We want to keep `policy-controller-validate` up-to-date. Taking a step back, the mirroring of OSS to ACM registry is sometthing that should be done by internal CD pipeline not here.

Reply:
`gcr.io/kpt-fn/gatekeeper:v0.1` has been manually mirrored to `config-management-release` GCR by Sandy (It could be Steve if I remember it wrongly) in Q3. He claimed that it should be fine to mirror the image manually. It doesn't worth the efforts to automate now. I guess we will do on-demand mirroring until someone said something in b/193589886.